This patch configures ncurses and host-ncurses so that tic uses symlinks
instead of hard links. Hard links across directories are not supported under
AFS. This patch is only required when building on AFS or another filesystem
with this limitation.

T. Straumann, 8/24/2013

*** package/ncurses/ncurses.mk.orig	2013-08-24 09:49:44.000000002 -0700
--- package/ncurses/ncurses.mk	2013-08-24 09:56:24.000000002 -0700
***************
*** 42,48 ****
  	--enable-const \
  	--enable-overwrite \
  	--enable-broken_linker \
! 	--disable-static
  
  ifneq ($(BR2_ENABLE_DEBUG),y)
  NCURSES_CONF_OPT += --without-debug
--- 42,49 ----
  	--enable-const \
  	--enable-overwrite \
  	--enable-broken_linker \
! 	--disable-static \
! 	--enable-symlinks
  
  ifneq ($(BR2_ENABLE_DEBUG),y)
  NCURSES_CONF_OPT += --without-debug
***************
*** 133,139 ****
  HOST_NCURSES_DEPENDENCIES =
  
  HOST_NCURSES_CONF_OPT = \
! 	--without-shared --without-gpm
  
  $(eval $(call AUTOTARGETS))
  $(eval $(call AUTOTARGETS,host))
--- 134,140 ----
  HOST_NCURSES_DEPENDENCIES =
  
  HOST_NCURSES_CONF_OPT = \
! 	--without-shared --without-gpm --enable-symlinks
  
  $(eval $(call AUTOTARGETS))
  $(eval $(call AUTOTARGETS,host))
